src/tests/unit/tests/views/content/__snapshots__/content-view.test.tsx.snap

```diff
@@ -1,31 +1,27 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`content view renders 1`] = `
-<Page
-  deps={
-    {
-      "textContent": {
-        "applicationTitle": "THE_APPLICATION_TITLE",
-      },
-    }
-  }
->
-  <div
-    className="content-container"
+<DocumentFragment>
+  <mock-page
+    deps="[object Object]"
```

Comment:
here aswell can we try expectMockedComponentPropsToMatchSnapshots(Page) if possible so that we can see deps values object

Reply:
Yes, implemented suggested changes.